editoast/src/client/import_rolling_stock.rs

```diff
@@ -70,162 +76,251 @@ pub async fn import_rolling_stock(
     Ok(())
 }
 
+pub async fn import_towed_rolling_stock(
+    args: ImportRollingStockArgs,
+    db_pool: Arc<DbConnectionPoolV2>,
+) -> Result<(), Box<dyn Error + Send + Sync>> {
+    for towed_rolling_stock_path in args.rolling_stock_path {
+        let towed_rolling_stock_file = File::open(towed_rolling_stock_path)?;
+        let towed_rolling_stock_form: TowedRollingStock =
```

Comment:
```suggestion
        let towed_rolling_stock: TowedRollingStock =
```

We conventionally use the `-form` suffix to represent what the frontend sends in the http api. Since we're using the schema directly here (rightfully so), the suffix may be misleading, wdyt?

Reply:
Done in [bfa4ce5](https://github.com/OpenRailAssociation/osrd/pull/9580/commits/bfa4ce50e951609623abde6fe570b979a579c0a9).